Editorial updates (#205)
w3c/wai-website-theme: _style-guide/structure-presentation.md

##### Semicolon (optional)

> If the information is short and simple, you can use a semicolon to combine two closely related ideas in one sentence. The text before and the text after the semicolon should both be grammatically complete sentences.

Reviewer: ```suggestion
For short and simple information, you can use a semicolon to combine closely related ideas. The text before and the text after the semicolon should both be grammatically complete sentences.
```

Simpler to read without losing anything of the understanding.

Author: I've taken your suggestion to shorten the first clause, but I've kept the active voice in the second clause because we generally try to avoid passive constructions where possible. 
I also think it's important to retain the word "closely". One of the key characteristics of semicolon usage is that it joins ideas that are more closely related than ideas that would typically be separated by a full stop.

Reviewer: I would still be inclined to remove 'closely'. Few people will use a semicolon; those that do will likely be ok with their use. Those that aren't will ask 'how close is closely? what does that mean?'. That is a definitional rabbit hole we can avoid by removing the word.

Although I had not suggested removing 'closely'. However, thinking about it, I would.

I think I would be inclined to:

```suggestion
If the information is short and simple, you can use a semicolon to combine two related ideas. The text before and the text after the semicolon should both be grammatically complete sentences.
```

That is short and simple, and avoids unnecessary head scratching.
